tools/filters/join.py

```diff
@@ -284,7 +285,7 @@ def fill_empty_columns( line, split, fill_values ):
     return split.join( filled_columns )
 
 
-def join_files( filename1, column1, filename2, column2, out_filename, split=None, buffer=1000000, keep_unmatched=False, keep_partial=False, index_depth=3, fill_options=None ):
+def join_files( filename1, column1, filename2, column2, out_filename, split=None, buffer=1000000, keep_unmatched=False, keep_partial=False, keep_headers=False, index_depth=3, fill_options=None ):
```

Comment:
this file needs merging against dev so it respects our new whitespace requirements :/